integration_tests/test_ibc_rly.py

```diff
@@ -44,27 +42,14 @@
 @pytest.fixture(scope="module")
 def ibc(request, tmp_path_factory):
     "prepare-network"
-    path = tmp_path_factory.mktemp("ibc_rly")
-    procs = []
-    try:
-        for network in prepare_network(
-            path,
-            "ibc",
-            relayer=cluster.Relayer.RLY.value,
-        ):
-            if network.proc:
-                procs.append(network.proc)
-                print("append:", network.proc)
-            yield network
-    finally:
-        print("finally:", procs)
-        for proc in procs:
-            try:
-                os.killpg(os.getpgid(proc.pid), signal.SIGTERM)
-            except ProcessLookupError:
-                pass
-            # proc.terminate()
-            proc.wait()
+    name = "ibc_rly"
+    path = tmp_path_factory.mktemp(name)
+    network = prepare_network(
+        path,
+        name,
+        relayer=cluster.Relayer.RLY.value,
+    )
+    yield from network
```

Comment:
```suggestion
    yield from prepare_network(
        path,
        name,
        relayer=cluster.Relayer.RLY.value,
    )
```

`network` variable would be a generator instead of a normal object, I think it's slightly better to not naming is at all, or use a name imply generator.